pkgs/games/gzdoom/default.nix

```diff
@@ -0,0 +1,33 @@
+{stdenv, fetchFromGitHub, cmake, fmod, mesa, SDL2}:
+
+stdenv.mkDerivation {
+  name = "gzdoom-git";
+  src = fetchFromGitHub{
+    owner = "coelckers";
+    repo = "gzdoom";
+    rev = "a59824cd8897dea5dd452c31be1328415478f990";
+    sha256 = "1lg9dk5prn2bjmyznq941a862alljvfgbb42whbpg0vw9vhpikak";
+  };
+
+  buildInputs = [ cmake fmod mesa SDL2 ];
+
+  cmakeFlags = [ "-DFMOD_LIBRARY=${fmod}/lib/libfmodex.so" ];
+
+  preConfigure=''
+    sed s@gzdoom.pk3@$out/share/gzdoom.pk3@ -i src/version.h
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp gzdoom $out/bin
+    mkdir -p $out/share
+    cp gzdoom.pk3 $out/share
+  '';
+
+  meta = {
+    homepage = https://github.com/coelckers/gzdoom;
+    description = "Adds an OpenGL renderer to the ZDoom source port";
```

Comment:
This sounds like it's an add-on/plug-in to the ZDoom package, which I presume it's not.